CODEOWNERS

```diff
@@ -1 +1,9 @@
 /plugin/ @mattermost/toolkit
+
+/.github/workflows/channels-ci.yml @mattermost/web-platform
+/webapp/package.json @mattermost/web-platform
+/webapp/channels/package.json @mattermost/web-platform
+/webapp/Makefile @mattermost/web-platform
+/webapp/package-lock.json @mattermost/web-platform
+/webapp/platform/*/package.json @mattermost/web-platform
```

Comment:
Should it be only the package.json file or the whole platform folder?

Reply:
I think just the package.json is fine for now. That folder include things like the common components package which we aren't going to be writing most of the code for